lib/ts/controllers/s-expandable-control.ts

```diff
@@ -145,12 +147,14 @@
             // attribute; for checkable controls this also means setting the `is-collapsed` class
             this.element.setAttribute("aria-expanded", this.isCollapsed() ? "false" : "true");
             if (this.isCheckable) {
-                var cc = this.controlledCollapsible;
+                var cc = this.controlledCollapsibles;
                 if (cc) {
                     var expected = !this.isCollapsed();
-                    var actual = cc.classList.contains("is-expanded");
+                    var actual = cc.some(element => element.classList.contains("is-expanded"));
                     if (expected !== actual) {
```

Comment:
```suggestion
                    if (cc.some(element => element.classList.contains("is-expanded") !== expected)) {
```

We want to fix this if _any_ element doesn't match our expectations.  As implemented, it would work for collapsing if something is out of sync but not expanding.